File: Modul 6/Soal 2/PRAK502-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
    return nilai1 - nilai2;
}
int mutlak(int angka)
{
    return abs (angka);
}
int main()
{
    int a,b,c,d,Hasil;
    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);
    Hasil = hitung(a,b)+hitung(c,d);
    printf("%d",mutlak(Hasil));
    return 0;
}

File: Modul 6/Soal 2/PRAK502-2210817310006-Ahmad Muhsin Zein.py
def hitung (nilai1, nilai2):
    return nilai1 - nilai2
def mutlak (angka):
    return abs (angka)

a,c,b,d = input().split()
a,c,b,d = map(int,(a,c,b,d))
Hasil = hitung(a,b)+hitung(c,d)
print(mutlak(Hasil))
